api/openapi-spec/swagger.json

```diff
@@ -3767,6 +3767,22 @@
      }
     }
    },
+   "v1.HostBridge": {
+    "description": "Represents a network which the vmi should connect to via a bridge",
+    "required": [
+     "bridgeName"
+    ],
+    "properties": {
+     "bridgeName": {
```

Comment:
Correct me if I'm wrong. In the currently proposed Network API we split the part how to get interface to pod and how to connect VM to it. With your proposed API, we always create the bridge inside Pod? I agree with the nodeBridgeName part, but I think we should just get a veth pair and let the other part of API do whatever user wants (L2 passtrough or bridge with DHCP proxy).

I haven't read actual code yet, if we actually expose just the veth, the name `bridgeName` is confusing.

Reply:
In my opinion `podBridge` (removed for now) and `hostBridge` are primitives out of more thich have stand-alone value but also can act as primitives for connecting. So these two are not supposed to support everything.